{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import networkx as nx\n",
    "import numpy as np\n",
    "\n",
    "from networkx.algorithms import bipartite"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "from services.pdb import pdbfile, pdbmolecule, pdbligand, pfam, go_terms\n",
    "from services.uniprot import pdb2uniprot"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "def projected_graph(Graph):\n",
    "    nx.set_node_attributes(Graph, bipartite.color(Graph), name='color')\n",
    "\n",
    "    top = [i for i in Graph.nodes if Graph.nodes[i]['color'] == 1]\n",
    "    projected = bipartite.overlap_weighted_projected_graph(Graph, top)\n",
    "    return projected"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "pf = pd.read_csv('./data/pocket_feature_scores.csv', header=None, names=['pocket_0', 'pocket_1', 'weight'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "all_pockets = pf['pocket_0'].unique().tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "structures, ligands = zip( *[i.split('_') for i in all_pockets] )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "structures = list(set(structures))\n",
    "ligands = list(set(ligands))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3777 1333\n"
     ]
    }
   ],
   "source": [
    "print(len(structures), len(ligands))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "pdb_uniprot_mapping = pdb2uniprot(structures[:100])\n",
    "PUG = nx.from_pandas_edgelist(pdb_uniprot_mapping, source='From', target='To')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('3zly', '3zlw', {'weight': 1.0})]"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "proj = projected_graph(PUG)\n",
    "list(proj.edges(data=True))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "pdb_pfam_mapping = pfam(','.join(structures[:100]))\n",
    "pdb_pfam_df = pd.DataFrame(pdb_pfam_mapping)\n",
    "PFG = nx.from_pandas_edgelist(pdb_pfam_df, source='structureId', target='pfamAcc')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('4CY8', '3IHG', {'weight': 1.0}),\n",
       " ('3KN5', '3ZLY', {'weight': 1.0}),\n",
       " ('3KN5', '4I5H', {'weight': 1.0}),\n",
       " ('3KN5', '4DYM', {'weight': 1.0}),\n",
       " ('3KN5', '3T8O', {'weight': 0.5}),\n",
       " ('3KN5', '3ZLW', {'weight': 1.0}),\n",
       " ('3LST', '3REO', {'weight': 0.5}),\n",
       " ('4IAW', '3APW', {'weight': 1.0}),\n",
       " ('2C31', '4K9P', {'weight': 1.0}),\n",
       " ('5JY1', '4HP8', {'weight': 1.0}),\n",
       " ('4I5H', '3ZLY', {'weight': 1.0}),\n",
       " ('4I5H', '4DYM', {'weight': 1.0}),\n",
       " ('4I5H', '3T8O', {'weight': 0.5}),\n",
       " ('4I5H', '3ZLW', {'weight': 1.0}),\n",
       " ('2HZI', '4E5W', {'weight': 1.0}),\n",
       " ('2HZI', '2NRU', {'weight': 1.0}),\n",
       " ('2HZI', '3DQW', {'weight': 1.0}),\n",
       " ('2HZI', '2EVA', {'weight': 1.0}),\n",
       " ('2HZI', '2Y6O', {'weight': 1.0}),\n",
       " ('2HZI', '3VO3', {'weight': 1.0}),\n",
       " ('2HZI', '5CNO', {'weight': 1.0}),\n",
       " ('2HZI', '5E8X', {'weight': 1.0}),\n",
       " ('2HZI', '2B7A', {'weight': 1.0}),\n",
       " ('2HZI', '3C1X', {'weight': 1.0}),\n",
       " ('2B7A', '4E5W', {'weight': 1.0}),\n",
       " ('2B7A', '2NRU', {'weight': 1.0}),\n",
       " ('2B7A', '3DQW', {'weight': 1.0}),\n",
       " ('2B7A', '2EVA', {'weight': 1.0}),\n",
       " ('2B7A', '2Y6O', {'weight': 1.0}),\n",
       " ('2B7A', '3VO3', {'weight': 1.0}),\n",
       " ('2B7A', '5CNO', {'weight': 1.0}),\n",
       " ('2B7A', '5E8X', {'weight': 1.0}),\n",
       " ('2B7A', '3C1X', {'weight': 1.0}),\n",
       " ('3VO3', '4E5W', {'weight': 1.0}),\n",
       " ('3VO3', '2NRU', {'weight': 1.0}),\n",
       " ('3VO3', '3DQW', {'weight': 1.0}),\n",
       " ('3VO3', '2EVA', {'weight': 1.0}),\n",
       " ('3VO3', '2Y6O', {'weight': 1.0}),\n",
       " ('3VO3', '5CNO', {'weight': 1.0}),\n",
       " ('3VO3', '5E8X', {'weight': 1.0}),\n",
       " ('3VO3', '3C1X', {'weight': 1.0}),\n",
       " ('3FVU', '3ELE', {'weight': 1.0}),\n",
       " ('4MQY', '4J3D', {'weight': 1.0}),\n",
       " ('3T8O', '3ZLY', {'weight': 0.5}),\n",
       " ('3T8O', '4DYM', {'weight': 0.5}),\n",
       " ('3T8O', '3ZLW', {'weight': 0.5}),\n",
       " ('3RUC', '2X6T', {'weight': 1.0}),\n",
       " ('2Y6O', '4E5W', {'weight': 1.0}),\n",
       " ('2Y6O', '2NRU', {'weight': 1.0}),\n",
       " ('2Y6O', '3DQW', {'weight': 1.0}),\n",
       " ('2Y6O', '2EVA', {'weight': 1.0}),\n",
       " ('2Y6O', '5CNO', {'weight': 1.0}),\n",
       " ('2Y6O', '5E8X', {'weight': 1.0}),\n",
       " ('2Y6O', '3C1X', {'weight': 1.0}),\n",
       " ('3C1X', '4E5W', {'weight': 1.0}),\n",
       " ('3C1X', '2NRU', {'weight': 1.0}),\n",
       " ('3C1X', '3DQW', {'weight': 1.0}),\n",
       " ('3C1X', '2EVA', {'weight': 1.0}),\n",
       " ('3C1X', '5CNO', {'weight': 1.0}),\n",
       " ('3C1X', '5E8X', {'weight': 1.0}),\n",
       " ('2EVA', '4E5W', {'weight': 1.0}),\n",
       " ('2EVA', '2NRU', {'weight': 1.0}),\n",
       " ('2EVA', '3DQW', {'weight': 1.0}),\n",
       " ('2EVA', '5CNO', {'weight': 1.0}),\n",
       " ('2EVA', '5E8X', {'weight': 1.0}),\n",
       " ('4N5N', '4BMV', {'weight': 1.0}),\n",
       " ('2NRU', '4E5W', {'weight': 1.0}),\n",
       " ('2NRU', '3DQW', {'weight': 1.0}),\n",
       " ('2NRU', '5CNO', {'weight': 1.0}),\n",
       " ('2NRU', '5E8X', {'weight': 1.0}),\n",
       " ('3DQW', '4E5W', {'weight': 1.0}),\n",
       " ('3DQW', '5CNO', {'weight': 1.0}),\n",
       " ('3DQW', '5E8X', {'weight': 1.0}),\n",
       " ('3ZLY', '4DYM', {'weight': 1.0}),\n",
       " ('3ZLY', '3ZLW', {'weight': 1.0}),\n",
       " ('3NXU', '4KKY', {'weight': 1.0}),\n",
       " ('5E8X', '4E5W', {'weight': 1.0}),\n",
       " ('5E8X', '5CNO', {'weight': 1.0}),\n",
       " ('2CDU', '3HYW', {'weight': 0.5}),\n",
       " ('3ZLW', '4DYM', {'weight': 1.0}),\n",
       " ('4E5W', '5CNO', {'weight': 1.0})]"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "proj_2 = projected_graph(PFG)\n",
    "list(proj_2.edges(data=True))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "AtlasView({'3KN5': {'weight': 1.0}, '4I5H': {'weight': 1.0}, '3T8O': {'weight': 0.5}, '4DYM': {'weight': 1.0}, '3ZLW': {'weight': 1.0}})"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "proj_2['3zly'.upper()]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [],
   "source": [
    "test = go_terms(','.join(['3th5','2fju' ]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'id': 'GO:0001764',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'neuron migration',\n",
       "   'definition': 'The characteristic movement of an immature neuron from germinal zones to specific positions where they will reside as they mature.',\n",
       "   'synonyms': 'neuron chemotaxis, neuron guidance, neuronal migration',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0001934',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of protein phosphorylation',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of addition of phosphate groups to amino acids within a protein.',\n",
       "   'synonyms': 'activation of protein amino acid phosphorylation, positive regulation of protein amino acid phosphorylation, stimulation of protein amino acid phosphorylation, up regulation of protein amino acid phosphorylation, up-regulation of protein amino acid phosphorylation, upregulation of protein amino acid phosphorylation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0002551',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'mast cell chemotaxis',\n",
       "   'definition': 'The movement of a mast cell in response to an external stimulus.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0006935',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'chemotaxis',\n",
       "   'definition': 'The directed movement of a motile cell or organism, or the directed growth of a cell guided by a specific chemical concentration gradient. Movement may be towards a higher concentration (positive chemotaxis) or towards a lower concentration (negative chemotaxis).',\n",
       "   'synonyms': 'taxis in response to chemical stimulus',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0006954',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'inflammatory response',\n",
       "   'definition': 'The immediate defensive reaction (by vertebrate tissue) to infection or injury caused by chemical or physical agents. The process is characterized by local vasodilation, extravasation of plasma into intercellular spaces and accumulation of white blood cells and macrophages.',\n",
       "   'synonyms': 'Inflammation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007015',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'actin filament organization',\n",
       "   'definition': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of cytoskeletal structures comprising actin filaments. Includes processes that control the spatial distribution of actin filaments, such as organizing filaments into meshworks, bundles, or other structures, as by cross-linking.',\n",
       "   'synonyms': 'actin filament organisation, regulation of actin filament localization',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007155',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell adhesion',\n",
       "   'definition': 'The attachment of a cell, either to another cell or to an underlying substrate such as the extracellular matrix, via cell adhesion molecules.',\n",
       "   'synonyms': 'cell adhesion molecule activity, single organism cell adhesion',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007160',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell-matrix adhesion',\n",
       "   'definition': 'The binding of a cell to the extracellular matrix via adhesion molecules.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007163',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'establishment or maintenance of cell polarity',\n",
       "   'definition': 'Any cellular process that results in the specification, formation or maintenance of anisotropic intracellular organization or cell growth patterns.',\n",
       "   'synonyms': 'cell polarity, establishment and/or maintenance of cell polarity, establishment and/or maintenance of cell polarization',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007264',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'small GTPase mediated signal transduction',\n",
       "   'definition': 'Any series of molecular signals in which a small monomeric GTPase relays one or more of the signals.',\n",
       "   'synonyms': 'small GTPase-mediated signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007266',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Rho protein signal transduction',\n",
       "   'definition': 'A series of molecular signals within the cell that are mediated by a member of the Rho family of proteins switching to a GTP-bound active state.',\n",
       "   'synonyms': 'Rho mediated signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007596',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'blood coagulation',\n",
       "   'definition': 'The sequential process in which the multiple coagulation factors of the blood interact, ultimately resulting in the formation of an insoluble fibrin clot; it may be divided into three stages: stage 1, the formation of intrinsic and extrinsic prothrombin converting principle; stage 2, the formation of thrombin; stage 3, the formation of stable fibrin polymers.',\n",
       "   'synonyms': 'blood clotting',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0008045',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'motor neuron axon guidance',\n",
       "   'definition': 'The process in which the migration of an axon growth cone of a motor neuron is directed to a specific target site in response to a combination of attractive and repulsive cues.',\n",
       "   'synonyms': 'motoneuron axon guidance, motor axon guidance, motor axon pathfinding',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0008283',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell proliferation',\n",
       "   'definition': 'The multiplication or reproduction of cells, resulting in the expansion of a cell population.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0008360',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of cell shape',\n",
       "   'definition': 'Any process that modulates the surface configuration of a cell.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0008361',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of cell size',\n",
       "   'definition': 'Any process that modulates the size of a cell.',\n",
       "   'synonyms': 'cell size control',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0009611',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'response to wounding',\n",
       "   'definition': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating damage to the organism.',\n",
       "   'synonyms': 'physiological response to wounding',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0009653',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'anatomical structure morphogenesis',\n",
       "   'definition': 'The process in which anatomical structures are generated and organized. Morphogenesis pertains to the creation of form.',\n",
       "   'synonyms': 'anatomical structure organization, embryogenesis and morphogenesis, morphogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0010310',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of hydrogen peroxide metabolic process',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving hydrogen peroxide.',\n",
       "   'synonyms': 'regulation of hydrogen peroxide metabolism',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0010591',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of lamellipodium assembly',\n",
       "   'definition': 'Any process that modulates the rate, frequency or extent of the formation of a lamellipodium, a thin sheetlike extension of the surface of a migrating cell.',\n",
       "   'synonyms': 'regulation of lamellipodium biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0010592',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of lamellipodium assembly',\n",
       "   'definition': 'Any process that increases the rate, frequency or extent of the formation of a lamellipodium, a thin sheetlike extension of the surface of a migrating cell.',\n",
       "   'synonyms': 'positive regulation of lamellipodium biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0010811',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of cell-substrate adhesion',\n",
       "   'definition': 'Any process that increases the frequency, rate or extent of cell-substrate adhesion. Cell-substrate adhesion is the attachment of a cell to the underlying substrate via adhesion molecules.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0016601',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Rac protein signal transduction',\n",
       "   'definition': 'A series of molecular signals within the cell that are mediated by a member of the Rac family of proteins switching to a GTP-bound active state.',\n",
       "   'synonyms': 'Rac mediated signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030031',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell projection assembly',\n",
       "   'definition': 'Formation of a prolongation or process extending from a cell, e.g. a flagellum or axon.',\n",
       "   'synonyms': 'cell projection biogenesis, formation of a cell surface projection',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030032',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'lamellipodium assembly',\n",
       "   'definition': 'Formation of a lamellipodium, a thin sheetlike extension of the surface of a migrating cell.',\n",
       "   'synonyms': 'lamellipodium biogenesis, lamellipodium biosynthesis, lamellipodium formation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030036',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'actin cytoskeleton organization',\n",
       "   'definition': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of cytoskeletal structures comprising actin filaments and their associated proteins.',\n",
       "   'synonyms': 'actin cytoskeleton organisation, actin cytoskeleton organization and biogenesis, actin modulating activity',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030041',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'actin filament polymerization',\n",
       "   'definition': 'Assembly of actin filaments by the addition of actin monomers to a filament.',\n",
       "   'synonyms': 'actin polymerization, actin polymerizing activity',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030334',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of cell migration',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of cell migration.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030865',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cortical cytoskeleton organization',\n",
       "   'definition': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of cytoskeletal structures in the cell cortex, i.e. just beneath the plasma membrane.',\n",
       "   'synonyms': 'cortical cytoskeleton organisation, cortical cytoskeleton organization and biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0031116',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of microtubule polymerization',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of microtubule polymerization.',\n",
       "   'synonyms': 'activation of microtubule polymerization, stimulation of microtubule polymerization, up regulation of microtubule polymerization, up-regulation of microtubule polymerization, upregulation of microtubule polymerization',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0031295',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'T cell costimulation',\n",
       "   'definition': 'The process of providing, via surface-bound receptor-ligand pairs, a second, antigen-independent, signal in addition to that provided by the T cell receptor to augment T cell activation.',\n",
       "   'synonyms': 'T cell co-stimulation, T lymphocyte costimulation, T-cell co-stimulation, T-cell costimulation, T-lymphocyte costimulation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0031529',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'ruffle organization',\n",
       "   'definition': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of a ruffle, a projection at the leading edge of a crawling cell.',\n",
       "   'synonyms': 'ruffle organisation, ruffle organization and biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0032707',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'negative regulation of interleukin-23 production',\n",
       "   'definition': 'Any process that stops, prevents, or reduces the frequency, rate, or extent of interleukin-23 production.',\n",
       "   'synonyms': 'down regulation of interleukin-23 production, down-regulation of interleukin-23 production, downregulation of interleukin-23 production, inhibition of interleukin-23 production, negative regulation of IL-23 production',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0032956',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of actin cytoskeleton organization',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of the formation, arrangement of constituent parts, or disassembly of cytoskeletal structures comprising actin filaments and their associated proteins.',\n",
       "   'synonyms': 'regulation of actin cytoskeleton organisation, regulation of actin cytoskeleton organization and biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0034446',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'substrate adhesion-dependent cell spreading',\n",
       "   'definition': 'The morphogenetic process that results in flattening of a cell as a consequence of its adhesion to a substrate.',\n",
       "   'synonyms': 'cell spreading during cell substrate adhesion, substrate adhesion dependent cell spreading',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0035025',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of Rho protein signal transduction',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of Rho protein signal transduction.',\n",
       "   'synonyms': 'activation of Rho protein signal transduction, stimulation of Rho protein signal transduction, up regulation of Rho protein signal transduction, up-regulation of Rho protein signal transduction, upregulation of Rho protein signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0035556',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'intracellular signal transduction',\n",
       "   'definition': 'The process in which a signal is passed on to downstream components within the cell, which become activated themselves to further propagate the signal and finally trigger a change in the function or state of the cell.',\n",
       "   'synonyms': 'intracellular protein kinase cascade, intracellular signal transduction pathway, intracellular signaling cascade, intracellular signaling chain, intracellular signaling pathway, protein kinase cascade, signal transduction via intracellular signaling cascade, signal transmission via intracellular cascade',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0035774',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of insulin secretion involved in cellular response to glucose stimulus',\n",
       "   'definition': 'Any process that increases the frequency, rate or extent of the regulated release of insulin that contributes to the response of a cell to glucose.',\n",
       "   'synonyms': 'positive regulation of insulin secretion in response to glucose',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0038095',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Fc-epsilon receptor signaling pathway',\n",
       "   'definition': 'A series of molecular signals initiated by the binding of the Fc portion of immunoglobulin E (IgE) to an Fc-epsilon receptor on the surface of a signal-receiving cell, and ending with regulation of a downstream cellular process, e.g. transcription. The Fc portion of an immunoglobulin is its C-terminal constant region.',\n",
       "   'synonyms': 'Fc-epsilon receptor signalling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0038096',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Fc-gamma receptor signaling pathway involved in phagocytosis',\n",
       "   'definition': 'An Fc-gamma receptor signaling pathway that contributes to the endocytic engulfment of external particulate material by phagocytes.',\n",
       "   'synonyms': 'Fc gamma receptor-dependent phagocytosis, Fc-gamma receptor signalling pathway involved in phagocytosis, Fcgamma receptor-mediated phagocytosis, IgG-mediated phagocytosis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0043312',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'neutrophil degranulation',\n",
       "   'definition': 'The regulated exocytosis of secretory granules containing preformed mediators such as proteases, lipases, and inflammatory mediators by a neutrophil.',\n",
       "   'synonyms': 'heterophil degranulation, neutrophil granule exocytosis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0043652',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'engulfment of apoptotic cell',\n",
       "   'definition': 'The removal of the apoptotic cell by phagocytosis, by a neighboring cell or by a phagocyte.',\n",
       "   'synonyms': 'engulfment of apoptotic cell corpse, engulfment of cell corpse',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0045428',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of nitric oxide biosynthetic process',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of nitric oxide.',\n",
       "   'synonyms': 'regulation of nitric oxide anabolism, regulation of nitric oxide biosynthesis, regulation of nitric oxide formation, regulation of nitric oxide synthesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0045453',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'bone resorption',\n",
       "   'definition': 'The process in which specialized cells known as osteoclasts degrade the organic and inorganic portions of bone, and endocytose and transport the degradation products.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0045740',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of DNA replication',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of DNA replication.',\n",
       "   'synonyms': 'activation of DNA replication, stimulation of DNA replication, up regulation of DNA replication, up-regulation of DNA replication, upregulation of DNA replication',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048010',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'vascular endothelial growth factor receptor signaling pathway',\n",
       "   'definition': 'Any series of molecular signals initiated by the binding of an extracellular ligand to a vascular endothelial growth factor receptor (VEGFR) located on the surface of the receiving cell, and ending with regulation of a downstream cellular process, e.g. transcription.',\n",
       "   'synonyms': 'VEGF receptor signaling pathway, VEGF receptor signalling pathway, VEGFR signaling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048012',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'hepatocyte growth factor receptor signaling pathway',\n",
       "   'definition': 'The series of molecular signals generated as a consequence of the hepatocyte growth factor receptor binding to one of its physiological ligands.',\n",
       "   'synonyms': 'HGF receptor signaling pathway, HGF receptor signalling pathway, Met signaling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048013',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'ephrin receptor signaling pathway',\n",
       "   'definition': 'The series of molecular signals generated as a consequence of an ephrin receptor binding to an ephrin.',\n",
       "   'synonyms': 'Eph receptor signaling pathway, Eph receptor signalling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048261',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'negative regulation of receptor-mediated endocytosis',\n",
       "   'definition': 'Any process that stops, prevents, or reduces the frequency, rate or extent of receptor mediated endocytosis, the uptake of external materials by cells, utilizing receptors to ensure specificity of transport.',\n",
       "   'synonyms': 'down regulation of receptor mediated endocytosis, down-regulation of receptor mediated endocytosis, downregulation of receptor mediated endocytosis, inhibition of receptor mediated endocytosis, negative regulation of receptor mediated endocytosis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048870',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell motility',\n",
       "   'definition': 'Any process involved in the controlled self-propelled movement of a cell that results in translocation of the cell from one place to another.',\n",
       "   'synonyms': 'cell locomotion, cell movement, movement of a cell',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0050690',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of defense response to virus by virus',\n",
       "   'definition': 'Any viral process that modulates the frequency, rate, or extent of the antiviral response of the host cell or organism.',\n",
       "   'synonyms': 'regulation by virus of antiviral response, regulation of antiviral response by virus, viral regulation of antiviral response',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051056',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of small GTPase mediated signal transduction',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of small GTPase mediated signal transduction.',\n",
       "   'synonyms': 'regulation of small GTPase-mediated signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051492',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of stress fiber assembly',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of the assembly of a stress fiber, a bundle of microfilaments and other proteins found in fibroblasts.',\n",
       "   'synonyms': 'regulation of stress fibre biosynthesis, regulation of stress fibre formation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051496',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of stress fiber assembly',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of the assembly of a stress fiber, a bundle of microfilaments and other proteins found in fibroblasts.',\n",
       "   'synonyms': 'activation of stress fiber formation, positive regulation of stress fibre biosynthesis, positive regulation of stress fibre formation, stimulation of stress fiber formation, up regulation of stress fiber formation, up-regulation of stress fiber formation, upregulation of stress fiber formation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051668',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'localization within membrane',\n",
       "   'definition': 'Any process in which a substance or cellular entity, such as a protein complex or organelle, is transported to, and/or maintained in, a specific location within a membrane.',\n",
       "   'synonyms': 'establishment and maintenance of localization in membrane, establishment and maintenance of position in membrane, localisation within membrane, localization to membrane, positioning within membrane',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051894',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of focal adhesion assembly',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of focal adhesion assembly, the establishment and maturation of focal adhesions.',\n",
       "   'synonyms': 'activation of focal adhesion formation, stimulation of focal adhesion formation, up regulation of focal adhesion formation, up-regulation of focal adhesion formation, upregulation of focal adhesion formation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051897',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of protein kinase B signaling',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of protein kinase B signaling, a series of reactions mediated by the intracellular serine/threonine kinase protein kinase B.',\n",
       "   'synonyms': 'activation of protein kinase B signaling cascade, positive regulation of AKT signaling cascade, positive regulation of AKT signalling cascade, positive regulation of PKB signaling cascade, positive regulation of PKB signalling cascade, positive regulation of protein kinase B signaling cascade, positive regulation of protein kinase B signalling cascade, stimulation of protein kinase B signaling cascade, up regulation of protein kinase B signaling cascade, up-regulation of protein kinase B signaling cascade, upregulation of protein kinase B signaling cascade',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0060071',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Wnt signaling pathway, planar cell polarity pathway',\n",
       "   'definition': 'The series of molecular signals initiated by binding of a Wnt protein to a receptor on the surface of the target cell where activated receptors signal via downstream effectors including C-Jun N-terminal kinase (JNK) to modulate cytoskeletal elements and control cell polarity.',\n",
       "   'synonyms': 'PCP pathway, Wnt receptor signaling pathway, planar cell polarity pathway, Wnt receptor signalling pathway, planar cell polarity pathway, Wnt-JNK signaling pathway, Wnt-PCP signaling pathway, Wnt-activated signaling pathway, planar cell polarity pathway, non-canonical Wnt signaling pathway, planar cell polarity pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0060263',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of respiratory burst',\n",
       "   'definition': 'Any process that modulates the rate frequency or extent of a phase of elevated metabolic activity, during which oxygen consumption increases; this leads to the production, by an NADH dependent system, of hydrogen peroxide (H2O2), superoxide anions and hydroxyl radicals.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0071260',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cellular response to mechanical stimulus',\n",
       "   'definition': 'Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a mechanical stimulus.',\n",
       "   'synonyms': 'cellular mechanical stimulus response',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0071526',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'semaphorin-plexin signaling pathway',\n",
       "   'definition': 'A series of molecular signals generated as a consequence of a semaphorin receptor (composed of a plexin and a neurophilin) binding to a semaphorin ligand.',\n",
       "   'synonyms': 'semaphorin-plexin signalling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0090023',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of neutrophil chemotaxis',\n",
       "   'definition': 'Any process that increases the frequency, rate, or extent of neutrophil chemotaxis. Neutrophil chemotaxis is the directed movement of a neutrophil cell, the most numerous polymorphonuclear leukocyte found in the blood, in response to an external stimulus, usually an infection or wounding.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0097178',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'ruffle assembly',\n",
       "   'definition': 'The aggregation, arrangement and bonding together of a set of components to form a ruffle, a projection at the leading edge of a crawling cell; the protrusions are supported by a microfilament meshwork. The formation of ruffles (also called membrane ruffling) is thought to be controlled by a group of enzymes known as Rho GTPases, specifically RhoA, Rac1 and cdc42.',\n",
       "   'synonyms': 'membrane ruffle formation, membrane ruffling',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:1900026',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of substrate adhesion-dependent cell spreading',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of substrate adhesion-dependent cell spreading.',\n",
       "   'synonyms': 'positive regulation of cell spreading during cell substrate adhesion, positive regulation of substrate adhesion dependent cell spreading, up regulation of cell spreading during cell substrate adhesion, up regulation of substrate adhesion dependent cell spreading, up regulation of substrate adhesion-dependent cell spreading',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:1902622',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of neutrophil migration',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of neutrophil migration.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0000139',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Golgi membrane',\n",
       "   'definition': 'The lipid bilayer surrounding any of the compartments of the Golgi apparatus.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005737',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cytoplasm',\n",
       "   'definition': 'All of the contents of a cell excluding the plasma membrane and nucleus, but including other subcellular structures.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005789',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'endoplasmic reticulum membrane',\n",
       "   'definition': 'The lipid bilayer surrounding the endoplasmic reticulum.',\n",
       "   'synonyms': 'ER membrane',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005802',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'trans-Golgi network',\n",
       "   'definition': 'The network of interconnected tubular and cisternal structures located within the Golgi apparatus on the side distal to the endoplasmic reticulum, from which secretory vesicles emerge. The trans-Golgi network is important in the later stages of protein secretion where it is thought to play a key role in the sorting and targeting of secreted proteins to the correct destination.',\n",
       "   'synonyms': 'Golgi trans face, Golgi trans-face, TGN, late Golgi, maturing face, trans Golgi network, trans face',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005829',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cytosol',\n",
       "   'definition': 'The part of the cytoplasm that does not contain organelles but which does contain other particulate matter, such as protein complexes.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005856',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cytoskeleton',\n",
       "   'definition': 'Any of the various filamentous elements that form the internal framework of cells, and typically remain after treatment of the cells with mild detergent to remove membrane constituents and soluble components of the cytoplasm. The term embraces intermediate filaments, microfilaments, microtubules, the microtrabecular lattice, and other structures characterized by a polymeric filamentous nature and long-range order within the cell. The various elements of the cytoskeleton not only serve in the maintenance of cellular shape but also have roles in other cellular functions, including cellular movement, cell division, endocytosis, and movement of organelles.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005884',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'actin filament',\n",
       "   'definition': 'A filamentous structure formed of a two-stranded helical polymer of the protein actin and associated proteins. Actin filaments are a major component of the contractile apparatus of skeletal muscle and the microfilaments of the cytoskeleton of eukaryotic cells. The filaments, comprising polymerized globular actin molecules, appear as flexible structures with a diameter of 5-9 nm. They are organized into a variety of linear bundles, two-dimensional networks, and three dimensional gels. In the cytoskeleton they are most highly concentrated in the cortex of the cell just beneath the plasma membrane.',\n",
       "   'synonyms': 'microfilament',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005886',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'plasma membrane',\n",
       "   'definition': 'The membrane surrounding a cell that separates the cell from its external environment. It consists of a phospholipid bilayer and associated proteins.',\n",
       "   'synonyms': 'bacterial inner membrane, cell membrane, cellular membrane, cytoplasmic membrane, inner endospore membrane, juxtamembrane, plasma membrane lipid bilayer, plasmalemma',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005925',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'focal adhesion',\n",
       "   'definition': 'Small region on the surface of a cell that anchors the cell to the extracellular matrix and that forms a point of termination of actin filaments.',\n",
       "   'synonyms': 'adhesion plaque, focal contact, hemi-adherens junction',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005938',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell cortex',\n",
       "   'definition': 'The region of a cell that lies just beneath the plasma membrane and often, but not always, contains a network of actin filaments and associated proteins.',\n",
       "   'synonyms': 'cell periphery, peripheral cytoplasm',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0016020',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'membrane',\n",
       "   'definition': 'A lipid bilayer along with all the proteins and protein complexes embedded in it an attached to it.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0030027',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'lamellipodium',\n",
       "   'definition': 'A thin sheetlike process extended by the leading edge of a migrating cell or extending cell process; contains a dense meshwork of actin filaments.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0030667',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'secretory granule membrane',\n",
       "   'definition': 'The lipid bilayer surrounding a secretory granule.',\n",
       "   'synonyms': 'secretory vesicle membrane',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0031410',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cytoplasmic vesicle',\n",
       "   'definition': 'A vesicle found in the cytoplasm of a cell.',\n",
       "   'synonyms': 'cytoplasmic membrane bounded vesicle, cytoplasmic membrane-enclosed vesicle, cytoplasmic, membrane-bounded vesicle',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0032587',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'ruffle membrane',\n",
       "   'definition': 'The portion of the plasma membrane surrounding a ruffle.',\n",
       "   'synonyms': 'membrane ruffle',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0036464',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cytoplasmic ribonucleoprotein granule',\n",
       "   'definition': 'A ribonucleoprotein granule located in the cytoplasm.',\n",
       "   'synonyms': 'Staufen granule',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0042470',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'melanosome',\n",
       "   'definition': 'A tissue-specific, membrane-bounded cytoplasmic organelle within which melanin pigments are synthesized and stored. Melanosomes are synthesized in melanocyte cells.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0042995',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell projection',\n",
       "   'definition': 'A prolongation or process extending from a cell, e.g. a flagellum or axon.',\n",
       "   'synonyms': 'cell process, cellular process, cellular projection',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0043197',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'dendritic spine',\n",
       "   'definition': 'A small, membranous protrusion from a dendrite that forms a postsynaptic compartment - typically receiving input from a single presynapse.  They function as partially isolated biochemical and an electrical compartments. Spine morphology is variable including \"thin\", \"stubby\", \"mushroom\", and \"branched\", with a continuum of intermediate morphologies. They typically terminate in a bulb shape, linked to the dendritic shaft by a restriction. Spine remodeling is though to be involved in synaptic plasticity.',\n",
       "   'synonyms': 'branched dendritic spine, dendrite spine, mushroom dendritic spine, sessile dendritic spine, stubby dendritic spine, thin dendritic spine',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0043231',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'intracellular membrane-bounded organelle',\n",
       "   'definition': 'Organized structure of distinctive morphology and function, bounded by a single or double lipid bilayer membrane and occurring within the cell. Includes the nucleus, mitochondria, plastids, vacuoles, and vesicles. Excludes the plasma membrane.',\n",
       "   'synonyms': 'intracellular membrane-enclosed organelle',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0055038',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'recycling endosome membrane',\n",
       "   'definition': 'The lipid bilayer surrounding a recycling endosome.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0070062',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'extracellular exosome',\n",
       "   'definition': 'A vesicle that is released into the extracellular region by fusion of the limiting endosomal membrane of a multivesicular body with the plasma membrane. Extracellular exosomes, also simply called exosomes, have a diameter of about 40-100 nm.',\n",
       "   'synonyms': 'exosome, extracellular vesicular exosome',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0098794',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'postsynapse',\n",
       "   'definition': 'The part of a synapse that is part of the post-synaptic cell.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0098978',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'glutamatergic synapse',\n",
       "   'definition': 'A synapse that uses glutamate as a neurotransmitter.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0101003',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'ficolin-1-rich granule membrane',\n",
       "   'definition': 'The lipid bilayer surrounding a ficolin-1-rich granule.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0000166',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'nucleotide binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with a nucleotide, any compound consisting of a nucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the ribose or deoxyribose.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0003924',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'GTPase activity',\n",
       "   'definition': 'Catalysis of the reaction: GTP + H2O = GDP + phosphate.',\n",
       "   'synonyms': 'ARF small monomeric GTPase activity, RHEB small monomeric GTPase activity, Rab small monomeric GTPase activity, Ran small monomeric GTPase activity, Ras small monomeric GTPase activity, Rho small monomeric GTPase activity, Sar small monomeric GTPase activity, dynamin GTPase activity, heterotrimeric G-protein GTPase activity, heterotrimeric G-protein GTPase, alpha-subunit, heterotrimeric G-protein GTPase, beta-subunit, heterotrimeric G-protein GTPase, gamma-subunit, hydrolase activity, acting on acid anhydrides, acting on GTP, involved in cellular and subcellular movement, protein-synthesizing GTPase activity, protein-synthesizing GTPase activity, elongation, protein-synthesizing GTPase activity, initiation, protein-synthesizing GTPase activity, termination, signal-recognition-particle GTPase activity, small monomeric GTPase activity, tubulin GTPase activity',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0005515',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'protein binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules).',\n",
       "   'synonyms': 'glycoprotein binding, protein amino acid binding',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0005525',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'GTP binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with GTP, guanosine triphosphate.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0016787',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'hydrolase activity',\n",
       "   'definition': 'Catalysis of the hydrolysis of various bonds, e.g. C-O, C-N, C-C, phosphoric anhydride bonds, etc. Hydrolase is the systematic name for any enzyme of EC class 3.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0017137',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Rab GTPase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with Rab protein, any member of the Rab subfamily of the Ras superfamily of monomeric GTPases.',\n",
       "   'synonyms': 'REP, Rab escort protein activity, Rab interactor activity',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0019899',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'enzyme binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with any enzyme.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0019901',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'protein kinase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with a protein kinase, any enzyme that catalyzes the transfer of a phosphate group, usually from ATP, to a protein substrate.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0031996',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'thioesterase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with any thioesterase enzyme.',\n",
       "   'synonyms': 'thiolesterase binding',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0042826',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'histone deacetylase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with the enzyme histone deacetylase.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0044877',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'protein-containing complex binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with a macromolecular complex.',\n",
       "   'synonyms': 'macromolecular complex binding, protein complex binding',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0051022',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Rho GDP-dissociation inhibitor binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with a Rho GDP-dissociation inhibitor protein.',\n",
       "   'synonyms': 'Rho GDI binding',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0051117',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'ATPase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with an ATPase, any enzyme that catalyzes the hydrolysis of ATP.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0001764',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'neuron migration',\n",
       "   'definition': 'The characteristic movement of an immature neuron from germinal zones to specific positions where they will reside as they mature.',\n",
       "   'synonyms': 'neuron chemotaxis, neuron guidance, neuronal migration',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0001934',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'positive regulation of protein phosphorylation',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of addition of phosphate groups to amino acids within a protein.',\n",
       "   'synonyms': 'activation of protein amino acid phosphorylation, positive regulation of protein amino acid phosphorylation, stimulation of protein amino acid phosphorylation, up regulation of protein amino acid phosphorylation, up-regulation of protein amino acid phosphorylation, upregulation of protein amino acid phosphorylation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0002551',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'mast cell chemotaxis',\n",
       "   'definition': 'The movement of a mast cell in response to an external stimulus.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0006935',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'chemotaxis',\n",
       "   'definition': 'The directed movement of a motile cell or organism, or the directed growth of a cell guided by a specific chemical concentration gradient. Movement may be towards a higher concentration (positive chemotaxis) or towards a lower concentration (negative chemotaxis).',\n",
       "   'synonyms': 'taxis in response to chemical stimulus',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0006954',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'inflammatory response',\n",
       "   'definition': 'The immediate defensive reaction (by vertebrate tissue) to infection or injury caused by chemical or physical agents. The process is characterized by local vasodilation, extravasation of plasma into intercellular spaces and accumulation of white blood cells and macrophages.',\n",
       "   'synonyms': 'Inflammation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007015',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'actin filament organization',\n",
       "   'definition': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of cytoskeletal structures comprising actin filaments. Includes processes that control the spatial distribution of actin filaments, such as organizing filaments into meshworks, bundles, or other structures, as by cross-linking.',\n",
       "   'synonyms': 'actin filament organisation, regulation of actin filament localization',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007155',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cell adhesion',\n",
       "   'definition': 'The attachment of a cell, either to another cell or to an underlying substrate such as the extracellular matrix, via cell adhesion molecules.',\n",
       "   'synonyms': 'cell adhesion molecule activity, single organism cell adhesion',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007160',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cell-matrix adhesion',\n",
       "   'definition': 'The binding of a cell to the extracellular matrix via adhesion molecules.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007163',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'establishment or maintenance of cell polarity',\n",
       "   'definition': 'Any cellular process that results in the specification, formation or maintenance of anisotropic intracellular organization or cell growth patterns.',\n",
       "   'synonyms': 'cell polarity, establishment and/or maintenance of cell polarity, establishment and/or maintenance of cell polarization',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007264',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'small GTPase mediated signal transduction',\n",
       "   'definition': 'Any series of molecular signals in which a small monomeric GTPase relays one or more of the signals.',\n",
       "   'synonyms': 'small GTPase-mediated signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007266',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'Rho protein signal transduction',\n",
       "   'definition': 'A series of molecular signals within the cell that are mediated by a member of the Rho family of proteins switching to a GTP-bound active state.',\n",
       "   'synonyms': 'Rho mediated signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007596',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'blood coagulation',\n",
       "   'definition': 'The sequential process in which the multiple coagulation factors of the blood interact, ultimately resulting in the formation of an insoluble fibrin clot; it may be divided into three stages: stage 1, the formation of intrinsic and extrinsic prothrombin converting principle; stage 2, the formation of thrombin; stage 3, the formation of stable fibrin polymers.',\n",
       "   'synonyms': 'blood clotting',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0008045',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'motor neuron axon guidance',\n",
       "   'definition': 'The process in which the migration of an axon growth cone of a motor neuron is directed to a specific target site in response to a combination of attractive and repulsive cues.',\n",
       "   'synonyms': 'motoneuron axon guidance, motor axon guidance, motor axon pathfinding',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0008283',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cell proliferation',\n",
       "   'definition': 'The multiplication or reproduction of cells, resulting in the expansion of a cell population.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0008360',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'regulation of cell shape',\n",
       "   'definition': 'Any process that modulates the surface configuration of a cell.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0008361',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'regulation of cell size',\n",
       "   'definition': 'Any process that modulates the size of a cell.',\n",
       "   'synonyms': 'cell size control',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0009611',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'response to wounding',\n",
       "   'definition': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating damage to the organism.',\n",
       "   'synonyms': 'physiological response to wounding',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0009653',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'anatomical structure morphogenesis',\n",
       "   'definition': 'The process in which anatomical structures are generated and organized. Morphogenesis pertains to the creation of form.',\n",
       "   'synonyms': 'anatomical structure organization, embryogenesis and morphogenesis, morphogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0010310',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'regulation of hydrogen peroxide metabolic process',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving hydrogen peroxide.',\n",
       "   'synonyms': 'regulation of hydrogen peroxide metabolism',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0010591',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'regulation of lamellipodium assembly',\n",
       "   'definition': 'Any process that modulates the rate, frequency or extent of the formation of a lamellipodium, a thin sheetlike extension of the surface of a migrating cell.',\n",
       "   'synonyms': 'regulation of lamellipodium biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0010592',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'positive regulation of lamellipodium assembly',\n",
       "   'definition': 'Any process that increases the rate, frequency or extent of the formation of a lamellipodium, a thin sheetlike extension of the surface of a migrating cell.',\n",
       "   'synonyms': 'positive regulation of lamellipodium biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0010811',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'positive regulation of cell-substrate adhesion',\n",
       "   'definition': 'Any process that increases the frequency, rate or extent of cell-substrate adhesion. Cell-substrate adhesion is the attachment of a cell to the underlying substrate via adhesion molecules.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0016601',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'Rac protein signal transduction',\n",
       "   'definition': 'A series of molecular signals within the cell that are mediated by a member of the Rac family of proteins switching to a GTP-bound active state.',\n",
       "   'synonyms': 'Rac mediated signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030031',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cell projection assembly',\n",
       "   'definition': 'Formation of a prolongation or process extending from a cell, e.g. a flagellum or axon.',\n",
       "   'synonyms': 'cell projection biogenesis, formation of a cell surface projection',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030032',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'lamellipodium assembly',\n",
       "   'definition': 'Formation of a lamellipodium, a thin sheetlike extension of the surface of a migrating cell.',\n",
       "   'synonyms': 'lamellipodium biogenesis, lamellipodium biosynthesis, lamellipodium formation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030036',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'actin cytoskeleton organization',\n",
       "   'definition': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of cytoskeletal structures comprising actin filaments and their associated proteins.',\n",
       "   'synonyms': 'actin cytoskeleton organisation, actin cytoskeleton organization and biogenesis, actin modulating activity',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030041',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'actin filament polymerization',\n",
       "   'definition': 'Assembly of actin filaments by the addition of actin monomers to a filament.',\n",
       "   'synonyms': 'actin polymerization, actin polymerizing activity',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030334',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'regulation of cell migration',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of cell migration.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030865',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cortical cytoskeleton organization',\n",
       "   'definition': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of cytoskeletal structures in the cell cortex, i.e. just beneath the plasma membrane.',\n",
       "   'synonyms': 'cortical cytoskeleton organisation, cortical cytoskeleton organization and biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0031116',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'positive regulation of microtubule polymerization',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of microtubule polymerization.',\n",
       "   'synonyms': 'activation of microtubule polymerization, stimulation of microtubule polymerization, up regulation of microtubule polymerization, up-regulation of microtubule polymerization, upregulation of microtubule polymerization',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0031295',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'T cell costimulation',\n",
       "   'definition': 'The process of providing, via surface-bound receptor-ligand pairs, a second, antigen-independent, signal in addition to that provided by the T cell receptor to augment T cell activation.',\n",
       "   'synonyms': 'T cell co-stimulation, T lymphocyte costimulation, T-cell co-stimulation, T-cell costimulation, T-lymphocyte costimulation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0031529',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'ruffle organization',\n",
       "   'definition': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of a ruffle, a projection at the leading edge of a crawling cell.',\n",
       "   'synonyms': 'ruffle organisation, ruffle organization and biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0032707',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'negative regulation of interleukin-23 production',\n",
       "   'definition': 'Any process that stops, prevents, or reduces the frequency, rate, or extent of interleukin-23 production.',\n",
       "   'synonyms': 'down regulation of interleukin-23 production, down-regulation of interleukin-23 production, downregulation of interleukin-23 production, inhibition of interleukin-23 production, negative regulation of IL-23 production',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0032956',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'regulation of actin cytoskeleton organization',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of the formation, arrangement of constituent parts, or disassembly of cytoskeletal structures comprising actin filaments and their associated proteins.',\n",
       "   'synonyms': 'regulation of actin cytoskeleton organisation, regulation of actin cytoskeleton organization and biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0034446',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'substrate adhesion-dependent cell spreading',\n",
       "   'definition': 'The morphogenetic process that results in flattening of a cell as a consequence of its adhesion to a substrate.',\n",
       "   'synonyms': 'cell spreading during cell substrate adhesion, substrate adhesion dependent cell spreading',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0035025',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'positive regulation of Rho protein signal transduction',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of Rho protein signal transduction.',\n",
       "   'synonyms': 'activation of Rho protein signal transduction, stimulation of Rho protein signal transduction, up regulation of Rho protein signal transduction, up-regulation of Rho protein signal transduction, upregulation of Rho protein signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0035556',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'intracellular signal transduction',\n",
       "   'definition': 'The process in which a signal is passed on to downstream components within the cell, which become activated themselves to further propagate the signal and finally trigger a change in the function or state of the cell.',\n",
       "   'synonyms': 'intracellular protein kinase cascade, intracellular signal transduction pathway, intracellular signaling cascade, intracellular signaling chain, intracellular signaling pathway, protein kinase cascade, signal transduction via intracellular signaling cascade, signal transmission via intracellular cascade',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0035774',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'positive regulation of insulin secretion involved in cellular response to glucose stimulus',\n",
       "   'definition': 'Any process that increases the frequency, rate or extent of the regulated release of insulin that contributes to the response of a cell to glucose.',\n",
       "   'synonyms': 'positive regulation of insulin secretion in response to glucose',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0038095',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'Fc-epsilon receptor signaling pathway',\n",
       "   'definition': 'A series of molecular signals initiated by the binding of the Fc portion of immunoglobulin E (IgE) to an Fc-epsilon receptor on the surface of a signal-receiving cell, and ending with regulation of a downstream cellular process, e.g. transcription. The Fc portion of an immunoglobulin is its C-terminal constant region.',\n",
       "   'synonyms': 'Fc-epsilon receptor signalling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0038096',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'Fc-gamma receptor signaling pathway involved in phagocytosis',\n",
       "   'definition': 'An Fc-gamma receptor signaling pathway that contributes to the endocytic engulfment of external particulate material by phagocytes.',\n",
       "   'synonyms': 'Fc gamma receptor-dependent phagocytosis, Fc-gamma receptor signalling pathway involved in phagocytosis, Fcgamma receptor-mediated phagocytosis, IgG-mediated phagocytosis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0043312',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'neutrophil degranulation',\n",
       "   'definition': 'The regulated exocytosis of secretory granules containing preformed mediators such as proteases, lipases, and inflammatory mediators by a neutrophil.',\n",
       "   'synonyms': 'heterophil degranulation, neutrophil granule exocytosis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0043652',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'engulfment of apoptotic cell',\n",
       "   'definition': 'The removal of the apoptotic cell by phagocytosis, by a neighboring cell or by a phagocyte.',\n",
       "   'synonyms': 'engulfment of apoptotic cell corpse, engulfment of cell corpse',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0045428',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'regulation of nitric oxide biosynthetic process',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of nitric oxide.',\n",
       "   'synonyms': 'regulation of nitric oxide anabolism, regulation of nitric oxide biosynthesis, regulation of nitric oxide formation, regulation of nitric oxide synthesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0045453',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'bone resorption',\n",
       "   'definition': 'The process in which specialized cells known as osteoclasts degrade the organic and inorganic portions of bone, and endocytose and transport the degradation products.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0045740',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'positive regulation of DNA replication',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of DNA replication.',\n",
       "   'synonyms': 'activation of DNA replication, stimulation of DNA replication, up regulation of DNA replication, up-regulation of DNA replication, upregulation of DNA replication',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048010',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'vascular endothelial growth factor receptor signaling pathway',\n",
       "   'definition': 'Any series of molecular signals initiated by the binding of an extracellular ligand to a vascular endothelial growth factor receptor (VEGFR) located on the surface of the receiving cell, and ending with regulation of a downstream cellular process, e.g. transcription.',\n",
       "   'synonyms': 'VEGF receptor signaling pathway, VEGF receptor signalling pathway, VEGFR signaling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048012',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'hepatocyte growth factor receptor signaling pathway',\n",
       "   'definition': 'The series of molecular signals generated as a consequence of the hepatocyte growth factor receptor binding to one of its physiological ligands.',\n",
       "   'synonyms': 'HGF receptor signaling pathway, HGF receptor signalling pathway, Met signaling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048013',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'ephrin receptor signaling pathway',\n",
       "   'definition': 'The series of molecular signals generated as a consequence of an ephrin receptor binding to an ephrin.',\n",
       "   'synonyms': 'Eph receptor signaling pathway, Eph receptor signalling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048261',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'negative regulation of receptor-mediated endocytosis',\n",
       "   'definition': 'Any process that stops, prevents, or reduces the frequency, rate or extent of receptor mediated endocytosis, the uptake of external materials by cells, utilizing receptors to ensure specificity of transport.',\n",
       "   'synonyms': 'down regulation of receptor mediated endocytosis, down-regulation of receptor mediated endocytosis, downregulation of receptor mediated endocytosis, inhibition of receptor mediated endocytosis, negative regulation of receptor mediated endocytosis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048870',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cell motility',\n",
       "   'definition': 'Any process involved in the controlled self-propelled movement of a cell that results in translocation of the cell from one place to another.',\n",
       "   'synonyms': 'cell locomotion, cell movement, movement of a cell',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0050690',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'regulation of defense response to virus by virus',\n",
       "   'definition': 'Any viral process that modulates the frequency, rate, or extent of the antiviral response of the host cell or organism.',\n",
       "   'synonyms': 'regulation by virus of antiviral response, regulation of antiviral response by virus, viral regulation of antiviral response',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051056',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'regulation of small GTPase mediated signal transduction',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of small GTPase mediated signal transduction.',\n",
       "   'synonyms': 'regulation of small GTPase-mediated signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051492',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'regulation of stress fiber assembly',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of the assembly of a stress fiber, a bundle of microfilaments and other proteins found in fibroblasts.',\n",
       "   'synonyms': 'regulation of stress fibre biosynthesis, regulation of stress fibre formation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051496',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'positive regulation of stress fiber assembly',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of the assembly of a stress fiber, a bundle of microfilaments and other proteins found in fibroblasts.',\n",
       "   'synonyms': 'activation of stress fiber formation, positive regulation of stress fibre biosynthesis, positive regulation of stress fibre formation, stimulation of stress fiber formation, up regulation of stress fiber formation, up-regulation of stress fiber formation, upregulation of stress fiber formation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051668',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'localization within membrane',\n",
       "   'definition': 'Any process in which a substance or cellular entity, such as a protein complex or organelle, is transported to, and/or maintained in, a specific location within a membrane.',\n",
       "   'synonyms': 'establishment and maintenance of localization in membrane, establishment and maintenance of position in membrane, localisation within membrane, localization to membrane, positioning within membrane',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051894',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'positive regulation of focal adhesion assembly',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of focal adhesion assembly, the establishment and maturation of focal adhesions.',\n",
       "   'synonyms': 'activation of focal adhesion formation, stimulation of focal adhesion formation, up regulation of focal adhesion formation, up-regulation of focal adhesion formation, upregulation of focal adhesion formation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051897',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'positive regulation of protein kinase B signaling',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of protein kinase B signaling, a series of reactions mediated by the intracellular serine/threonine kinase protein kinase B.',\n",
       "   'synonyms': 'activation of protein kinase B signaling cascade, positive regulation of AKT signaling cascade, positive regulation of AKT signalling cascade, positive regulation of PKB signaling cascade, positive regulation of PKB signalling cascade, positive regulation of protein kinase B signaling cascade, positive regulation of protein kinase B signalling cascade, stimulation of protein kinase B signaling cascade, up regulation of protein kinase B signaling cascade, up-regulation of protein kinase B signaling cascade, upregulation of protein kinase B signaling cascade',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0060071',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'Wnt signaling pathway, planar cell polarity pathway',\n",
       "   'definition': 'The series of molecular signals initiated by binding of a Wnt protein to a receptor on the surface of the target cell where activated receptors signal via downstream effectors including C-Jun N-terminal kinase (JNK) to modulate cytoskeletal elements and control cell polarity.',\n",
       "   'synonyms': 'PCP pathway, Wnt receptor signaling pathway, planar cell polarity pathway, Wnt receptor signalling pathway, planar cell polarity pathway, Wnt-JNK signaling pathway, Wnt-PCP signaling pathway, Wnt-activated signaling pathway, planar cell polarity pathway, non-canonical Wnt signaling pathway, planar cell polarity pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0060263',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'regulation of respiratory burst',\n",
       "   'definition': 'Any process that modulates the rate frequency or extent of a phase of elevated metabolic activity, during which oxygen consumption increases; this leads to the production, by an NADH dependent system, of hydrogen peroxide (H2O2), superoxide anions and hydroxyl radicals.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0071260',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cellular response to mechanical stimulus',\n",
       "   'definition': 'Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a mechanical stimulus.',\n",
       "   'synonyms': 'cellular mechanical stimulus response',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0071526',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'semaphorin-plexin signaling pathway',\n",
       "   'definition': 'A series of molecular signals generated as a consequence of a semaphorin receptor (composed of a plexin and a neurophilin) binding to a semaphorin ligand.',\n",
       "   'synonyms': 'semaphorin-plexin signalling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0090023',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'positive regulation of neutrophil chemotaxis',\n",
       "   'definition': 'Any process that increases the frequency, rate, or extent of neutrophil chemotaxis. Neutrophil chemotaxis is the directed movement of a neutrophil cell, the most numerous polymorphonuclear leukocyte found in the blood, in response to an external stimulus, usually an infection or wounding.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0097178',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'ruffle assembly',\n",
       "   'definition': 'The aggregation, arrangement and bonding together of a set of components to form a ruffle, a projection at the leading edge of a crawling cell; the protrusions are supported by a microfilament meshwork. The formation of ruffles (also called membrane ruffling) is thought to be controlled by a group of enzymes known as Rho GTPases, specifically RhoA, Rac1 and cdc42.',\n",
       "   'synonyms': 'membrane ruffle formation, membrane ruffling',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:1900026',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'positive regulation of substrate adhesion-dependent cell spreading',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of substrate adhesion-dependent cell spreading.',\n",
       "   'synonyms': 'positive regulation of cell spreading during cell substrate adhesion, positive regulation of substrate adhesion dependent cell spreading, up regulation of cell spreading during cell substrate adhesion, up regulation of substrate adhesion dependent cell spreading, up regulation of substrate adhesion-dependent cell spreading',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:1902622',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'regulation of neutrophil migration',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of neutrophil migration.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0000139',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'Golgi membrane',\n",
       "   'definition': 'The lipid bilayer surrounding any of the compartments of the Golgi apparatus.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005737',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cytoplasm',\n",
       "   'definition': 'All of the contents of a cell excluding the plasma membrane and nucleus, but including other subcellular structures.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005789',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'endoplasmic reticulum membrane',\n",
       "   'definition': 'The lipid bilayer surrounding the endoplasmic reticulum.',\n",
       "   'synonyms': 'ER membrane',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005802',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'trans-Golgi network',\n",
       "   'definition': 'The network of interconnected tubular and cisternal structures located within the Golgi apparatus on the side distal to the endoplasmic reticulum, from which secretory vesicles emerge. The trans-Golgi network is important in the later stages of protein secretion where it is thought to play a key role in the sorting and targeting of secreted proteins to the correct destination.',\n",
       "   'synonyms': 'Golgi trans face, Golgi trans-face, TGN, late Golgi, maturing face, trans Golgi network, trans face',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005829',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cytosol',\n",
       "   'definition': 'The part of the cytoplasm that does not contain organelles but which does contain other particulate matter, such as protein complexes.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005856',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cytoskeleton',\n",
       "   'definition': 'Any of the various filamentous elements that form the internal framework of cells, and typically remain after treatment of the cells with mild detergent to remove membrane constituents and soluble components of the cytoplasm. The term embraces intermediate filaments, microfilaments, microtubules, the microtrabecular lattice, and other structures characterized by a polymeric filamentous nature and long-range order within the cell. The various elements of the cytoskeleton not only serve in the maintenance of cellular shape but also have roles in other cellular functions, including cellular movement, cell division, endocytosis, and movement of organelles.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005884',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'actin filament',\n",
       "   'definition': 'A filamentous structure formed of a two-stranded helical polymer of the protein actin and associated proteins. Actin filaments are a major component of the contractile apparatus of skeletal muscle and the microfilaments of the cytoskeleton of eukaryotic cells. The filaments, comprising polymerized globular actin molecules, appear as flexible structures with a diameter of 5-9 nm. They are organized into a variety of linear bundles, two-dimensional networks, and three dimensional gels. In the cytoskeleton they are most highly concentrated in the cortex of the cell just beneath the plasma membrane.',\n",
       "   'synonyms': 'microfilament',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005886',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'plasma membrane',\n",
       "   'definition': 'The membrane surrounding a cell that separates the cell from its external environment. It consists of a phospholipid bilayer and associated proteins.',\n",
       "   'synonyms': 'bacterial inner membrane, cell membrane, cellular membrane, cytoplasmic membrane, inner endospore membrane, juxtamembrane, plasma membrane lipid bilayer, plasmalemma',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005925',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'focal adhesion',\n",
       "   'definition': 'Small region on the surface of a cell that anchors the cell to the extracellular matrix and that forms a point of termination of actin filaments.',\n",
       "   'synonyms': 'adhesion plaque, focal contact, hemi-adherens junction',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005938',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cell cortex',\n",
       "   'definition': 'The region of a cell that lies just beneath the plasma membrane and often, but not always, contains a network of actin filaments and associated proteins.',\n",
       "   'synonyms': 'cell periphery, peripheral cytoplasm',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0016020',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'membrane',\n",
       "   'definition': 'A lipid bilayer along with all the proteins and protein complexes embedded in it an attached to it.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0030027',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'lamellipodium',\n",
       "   'definition': 'A thin sheetlike process extended by the leading edge of a migrating cell or extending cell process; contains a dense meshwork of actin filaments.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0030667',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'secretory granule membrane',\n",
       "   'definition': 'The lipid bilayer surrounding a secretory granule.',\n",
       "   'synonyms': 'secretory vesicle membrane',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0031410',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cytoplasmic vesicle',\n",
       "   'definition': 'A vesicle found in the cytoplasm of a cell.',\n",
       "   'synonyms': 'cytoplasmic membrane bounded vesicle, cytoplasmic membrane-enclosed vesicle, cytoplasmic, membrane-bounded vesicle',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0032587',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'ruffle membrane',\n",
       "   'definition': 'The portion of the plasma membrane surrounding a ruffle.',\n",
       "   'synonyms': 'membrane ruffle',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0036464',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cytoplasmic ribonucleoprotein granule',\n",
       "   'definition': 'A ribonucleoprotein granule located in the cytoplasm.',\n",
       "   'synonyms': 'Staufen granule',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0042470',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'melanosome',\n",
       "   'definition': 'A tissue-specific, membrane-bounded cytoplasmic organelle within which melanin pigments are synthesized and stored. Melanosomes are synthesized in melanocyte cells.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0042995',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'cell projection',\n",
       "   'definition': 'A prolongation or process extending from a cell, e.g. a flagellum or axon.',\n",
       "   'synonyms': 'cell process, cellular process, cellular projection',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0043197',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'dendritic spine',\n",
       "   'definition': 'A small, membranous protrusion from a dendrite that forms a postsynaptic compartment - typically receiving input from a single presynapse.  They function as partially isolated biochemical and an electrical compartments. Spine morphology is variable including \"thin\", \"stubby\", \"mushroom\", and \"branched\", with a continuum of intermediate morphologies. They typically terminate in a bulb shape, linked to the dendritic shaft by a restriction. Spine remodeling is though to be involved in synaptic plasticity.',\n",
       "   'synonyms': 'branched dendritic spine, dendrite spine, mushroom dendritic spine, sessile dendritic spine, stubby dendritic spine, thin dendritic spine',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0043231',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'intracellular membrane-bounded organelle',\n",
       "   'definition': 'Organized structure of distinctive morphology and function, bounded by a single or double lipid bilayer membrane and occurring within the cell. Includes the nucleus, mitochondria, plastids, vacuoles, and vesicles. Excludes the plasma membrane.',\n",
       "   'synonyms': 'intracellular membrane-enclosed organelle',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0055038',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'recycling endosome membrane',\n",
       "   'definition': 'The lipid bilayer surrounding a recycling endosome.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0070062',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'extracellular exosome',\n",
       "   'definition': 'A vesicle that is released into the extracellular region by fusion of the limiting endosomal membrane of a multivesicular body with the plasma membrane. Extracellular exosomes, also simply called exosomes, have a diameter of about 40-100 nm.',\n",
       "   'synonyms': 'exosome, extracellular vesicular exosome',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0098794',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'postsynapse',\n",
       "   'definition': 'The part of a synapse that is part of the post-synaptic cell.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0098978',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'glutamatergic synapse',\n",
       "   'definition': 'A synapse that uses glutamate as a neurotransmitter.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0101003',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'ficolin-1-rich granule membrane',\n",
       "   'definition': 'The lipid bilayer surrounding a ficolin-1-rich granule.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0000166',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'nucleotide binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with a nucleotide, any compound consisting of a nucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the ribose or deoxyribose.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0003924',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'GTPase activity',\n",
       "   'definition': 'Catalysis of the reaction: GTP + H2O = GDP + phosphate.',\n",
       "   'synonyms': 'ARF small monomeric GTPase activity, RHEB small monomeric GTPase activity, Rab small monomeric GTPase activity, Ran small monomeric GTPase activity, Ras small monomeric GTPase activity, Rho small monomeric GTPase activity, Sar small monomeric GTPase activity, dynamin GTPase activity, heterotrimeric G-protein GTPase activity, heterotrimeric G-protein GTPase, alpha-subunit, heterotrimeric G-protein GTPase, beta-subunit, heterotrimeric G-protein GTPase, gamma-subunit, hydrolase activity, acting on acid anhydrides, acting on GTP, involved in cellular and subcellular movement, protein-synthesizing GTPase activity, protein-synthesizing GTPase activity, elongation, protein-synthesizing GTPase activity, initiation, protein-synthesizing GTPase activity, termination, signal-recognition-particle GTPase activity, small monomeric GTPase activity, tubulin GTPase activity',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0005515',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'protein binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules).',\n",
       "   'synonyms': 'glycoprotein binding, protein amino acid binding',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0005525',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'GTP binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with GTP, guanosine triphosphate.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0016787',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'hydrolase activity',\n",
       "   'definition': 'Catalysis of the hydrolysis of various bonds, e.g. C-O, C-N, C-C, phosphoric anhydride bonds, etc. Hydrolase is the systematic name for any enzyme of EC class 3.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0017137',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'Rab GTPase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with Rab protein, any member of the Rab subfamily of the Ras superfamily of monomeric GTPases.',\n",
       "   'synonyms': 'REP, Rab escort protein activity, Rab interactor activity',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0019899',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'enzyme binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with any enzyme.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0019901',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'protein kinase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with a protein kinase, any enzyme that catalyzes the transfer of a phosphate group, usually from ATP, to a protein substrate.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0031996',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'thioesterase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with any thioesterase enzyme.',\n",
       "   'synonyms': 'thiolesterase binding',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0042826',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'histone deacetylase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with the enzyme histone deacetylase.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0044877',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'protein-containing complex binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with a macromolecular complex.',\n",
       "   'synonyms': 'macromolecular complex binding, protein complex binding',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0051022',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'Rho GDP-dissociation inhibitor binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with a Rho GDP-dissociation inhibitor protein.',\n",
       "   'synonyms': 'Rho GDI binding',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0051117',\n",
       "  'structureId': '3TH5',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'ATPase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with an ATPase, any enzyme that catalyzes the hydrolysis of ATP.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0001764',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'neuron migration',\n",
       "   'definition': 'The characteristic movement of an immature neuron from germinal zones to specific positions where they will reside as they mature.',\n",
       "   'synonyms': 'neuron chemotaxis, neuron guidance, neuronal migration',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0001934',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of protein phosphorylation',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of addition of phosphate groups to amino acids within a protein.',\n",
       "   'synonyms': 'activation of protein amino acid phosphorylation, positive regulation of protein amino acid phosphorylation, stimulation of protein amino acid phosphorylation, up regulation of protein amino acid phosphorylation, up-regulation of protein amino acid phosphorylation, upregulation of protein amino acid phosphorylation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0002551',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'mast cell chemotaxis',\n",
       "   'definition': 'The movement of a mast cell in response to an external stimulus.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0006935',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'chemotaxis',\n",
       "   'definition': 'The directed movement of a motile cell or organism, or the directed growth of a cell guided by a specific chemical concentration gradient. Movement may be towards a higher concentration (positive chemotaxis) or towards a lower concentration (negative chemotaxis).',\n",
       "   'synonyms': 'taxis in response to chemical stimulus',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0006954',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'inflammatory response',\n",
       "   'definition': 'The immediate defensive reaction (by vertebrate tissue) to infection or injury caused by chemical or physical agents. The process is characterized by local vasodilation, extravasation of plasma into intercellular spaces and accumulation of white blood cells and macrophages.',\n",
       "   'synonyms': 'Inflammation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007015',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'actin filament organization',\n",
       "   'definition': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of cytoskeletal structures comprising actin filaments. Includes processes that control the spatial distribution of actin filaments, such as organizing filaments into meshworks, bundles, or other structures, as by cross-linking.',\n",
       "   'synonyms': 'actin filament organisation, regulation of actin filament localization',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007155',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell adhesion',\n",
       "   'definition': 'The attachment of a cell, either to another cell or to an underlying substrate such as the extracellular matrix, via cell adhesion molecules.',\n",
       "   'synonyms': 'cell adhesion molecule activity, single organism cell adhesion',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007160',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell-matrix adhesion',\n",
       "   'definition': 'The binding of a cell to the extracellular matrix via adhesion molecules.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007163',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'establishment or maintenance of cell polarity',\n",
       "   'definition': 'Any cellular process that results in the specification, formation or maintenance of anisotropic intracellular organization or cell growth patterns.',\n",
       "   'synonyms': 'cell polarity, establishment and/or maintenance of cell polarity, establishment and/or maintenance of cell polarization',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007264',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'small GTPase mediated signal transduction',\n",
       "   'definition': 'Any series of molecular signals in which a small monomeric GTPase relays one or more of the signals.',\n",
       "   'synonyms': 'small GTPase-mediated signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007266',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Rho protein signal transduction',\n",
       "   'definition': 'A series of molecular signals within the cell that are mediated by a member of the Rho family of proteins switching to a GTP-bound active state.',\n",
       "   'synonyms': 'Rho mediated signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007596',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'blood coagulation',\n",
       "   'definition': 'The sequential process in which the multiple coagulation factors of the blood interact, ultimately resulting in the formation of an insoluble fibrin clot; it may be divided into three stages: stage 1, the formation of intrinsic and extrinsic prothrombin converting principle; stage 2, the formation of thrombin; stage 3, the formation of stable fibrin polymers.',\n",
       "   'synonyms': 'blood clotting',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0008045',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'motor neuron axon guidance',\n",
       "   'definition': 'The process in which the migration of an axon growth cone of a motor neuron is directed to a specific target site in response to a combination of attractive and repulsive cues.',\n",
       "   'synonyms': 'motoneuron axon guidance, motor axon guidance, motor axon pathfinding',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0008283',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell proliferation',\n",
       "   'definition': 'The multiplication or reproduction of cells, resulting in the expansion of a cell population.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0008360',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of cell shape',\n",
       "   'definition': 'Any process that modulates the surface configuration of a cell.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0008361',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of cell size',\n",
       "   'definition': 'Any process that modulates the size of a cell.',\n",
       "   'synonyms': 'cell size control',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0009611',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'response to wounding',\n",
       "   'definition': 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating damage to the organism.',\n",
       "   'synonyms': 'physiological response to wounding',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0009653',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'anatomical structure morphogenesis',\n",
       "   'definition': 'The process in which anatomical structures are generated and organized. Morphogenesis pertains to the creation of form.',\n",
       "   'synonyms': 'anatomical structure organization, embryogenesis and morphogenesis, morphogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0010310',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of hydrogen peroxide metabolic process',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving hydrogen peroxide.',\n",
       "   'synonyms': 'regulation of hydrogen peroxide metabolism',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0010591',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of lamellipodium assembly',\n",
       "   'definition': 'Any process that modulates the rate, frequency or extent of the formation of a lamellipodium, a thin sheetlike extension of the surface of a migrating cell.',\n",
       "   'synonyms': 'regulation of lamellipodium biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0010592',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of lamellipodium assembly',\n",
       "   'definition': 'Any process that increases the rate, frequency or extent of the formation of a lamellipodium, a thin sheetlike extension of the surface of a migrating cell.',\n",
       "   'synonyms': 'positive regulation of lamellipodium biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0010811',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of cell-substrate adhesion',\n",
       "   'definition': 'Any process that increases the frequency, rate or extent of cell-substrate adhesion. Cell-substrate adhesion is the attachment of a cell to the underlying substrate via adhesion molecules.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0016601',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Rac protein signal transduction',\n",
       "   'definition': 'A series of molecular signals within the cell that are mediated by a member of the Rac family of proteins switching to a GTP-bound active state.',\n",
       "   'synonyms': 'Rac mediated signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030031',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell projection assembly',\n",
       "   'definition': 'Formation of a prolongation or process extending from a cell, e.g. a flagellum or axon.',\n",
       "   'synonyms': 'cell projection biogenesis, formation of a cell surface projection',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030032',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'lamellipodium assembly',\n",
       "   'definition': 'Formation of a lamellipodium, a thin sheetlike extension of the surface of a migrating cell.',\n",
       "   'synonyms': 'lamellipodium biogenesis, lamellipodium biosynthesis, lamellipodium formation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030036',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'actin cytoskeleton organization',\n",
       "   'definition': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of cytoskeletal structures comprising actin filaments and their associated proteins.',\n",
       "   'synonyms': 'actin cytoskeleton organisation, actin cytoskeleton organization and biogenesis, actin modulating activity',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030041',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'actin filament polymerization',\n",
       "   'definition': 'Assembly of actin filaments by the addition of actin monomers to a filament.',\n",
       "   'synonyms': 'actin polymerization, actin polymerizing activity',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030334',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of cell migration',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of cell migration.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0030865',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cortical cytoskeleton organization',\n",
       "   'definition': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of cytoskeletal structures in the cell cortex, i.e. just beneath the plasma membrane.',\n",
       "   'synonyms': 'cortical cytoskeleton organisation, cortical cytoskeleton organization and biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0031116',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of microtubule polymerization',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of microtubule polymerization.',\n",
       "   'synonyms': 'activation of microtubule polymerization, stimulation of microtubule polymerization, up regulation of microtubule polymerization, up-regulation of microtubule polymerization, upregulation of microtubule polymerization',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0031295',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'T cell costimulation',\n",
       "   'definition': 'The process of providing, via surface-bound receptor-ligand pairs, a second, antigen-independent, signal in addition to that provided by the T cell receptor to augment T cell activation.',\n",
       "   'synonyms': 'T cell co-stimulation, T lymphocyte costimulation, T-cell co-stimulation, T-cell costimulation, T-lymphocyte costimulation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0031529',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'ruffle organization',\n",
       "   'definition': 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of a ruffle, a projection at the leading edge of a crawling cell.',\n",
       "   'synonyms': 'ruffle organisation, ruffle organization and biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0032707',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'negative regulation of interleukin-23 production',\n",
       "   'definition': 'Any process that stops, prevents, or reduces the frequency, rate, or extent of interleukin-23 production.',\n",
       "   'synonyms': 'down regulation of interleukin-23 production, down-regulation of interleukin-23 production, downregulation of interleukin-23 production, inhibition of interleukin-23 production, negative regulation of IL-23 production',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0032956',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of actin cytoskeleton organization',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of the formation, arrangement of constituent parts, or disassembly of cytoskeletal structures comprising actin filaments and their associated proteins.',\n",
       "   'synonyms': 'regulation of actin cytoskeleton organisation, regulation of actin cytoskeleton organization and biogenesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0034446',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'substrate adhesion-dependent cell spreading',\n",
       "   'definition': 'The morphogenetic process that results in flattening of a cell as a consequence of its adhesion to a substrate.',\n",
       "   'synonyms': 'cell spreading during cell substrate adhesion, substrate adhesion dependent cell spreading',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0035025',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of Rho protein signal transduction',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of Rho protein signal transduction.',\n",
       "   'synonyms': 'activation of Rho protein signal transduction, stimulation of Rho protein signal transduction, up regulation of Rho protein signal transduction, up-regulation of Rho protein signal transduction, upregulation of Rho protein signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0035556',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'intracellular signal transduction',\n",
       "   'definition': 'The process in which a signal is passed on to downstream components within the cell, which become activated themselves to further propagate the signal and finally trigger a change in the function or state of the cell.',\n",
       "   'synonyms': 'intracellular protein kinase cascade, intracellular signal transduction pathway, intracellular signaling cascade, intracellular signaling chain, intracellular signaling pathway, protein kinase cascade, signal transduction via intracellular signaling cascade, signal transmission via intracellular cascade',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0035774',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of insulin secretion involved in cellular response to glucose stimulus',\n",
       "   'definition': 'Any process that increases the frequency, rate or extent of the regulated release of insulin that contributes to the response of a cell to glucose.',\n",
       "   'synonyms': 'positive regulation of insulin secretion in response to glucose',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0038095',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Fc-epsilon receptor signaling pathway',\n",
       "   'definition': 'A series of molecular signals initiated by the binding of the Fc portion of immunoglobulin E (IgE) to an Fc-epsilon receptor on the surface of a signal-receiving cell, and ending with regulation of a downstream cellular process, e.g. transcription. The Fc portion of an immunoglobulin is its C-terminal constant region.',\n",
       "   'synonyms': 'Fc-epsilon receptor signalling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0038096',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Fc-gamma receptor signaling pathway involved in phagocytosis',\n",
       "   'definition': 'An Fc-gamma receptor signaling pathway that contributes to the endocytic engulfment of external particulate material by phagocytes.',\n",
       "   'synonyms': 'Fc gamma receptor-dependent phagocytosis, Fc-gamma receptor signalling pathway involved in phagocytosis, Fcgamma receptor-mediated phagocytosis, IgG-mediated phagocytosis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0043312',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'neutrophil degranulation',\n",
       "   'definition': 'The regulated exocytosis of secretory granules containing preformed mediators such as proteases, lipases, and inflammatory mediators by a neutrophil.',\n",
       "   'synonyms': 'heterophil degranulation, neutrophil granule exocytosis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0043652',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'engulfment of apoptotic cell',\n",
       "   'definition': 'The removal of the apoptotic cell by phagocytosis, by a neighboring cell or by a phagocyte.',\n",
       "   'synonyms': 'engulfment of apoptotic cell corpse, engulfment of cell corpse',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0045428',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of nitric oxide biosynthetic process',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of nitric oxide.',\n",
       "   'synonyms': 'regulation of nitric oxide anabolism, regulation of nitric oxide biosynthesis, regulation of nitric oxide formation, regulation of nitric oxide synthesis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0045453',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'bone resorption',\n",
       "   'definition': 'The process in which specialized cells known as osteoclasts degrade the organic and inorganic portions of bone, and endocytose and transport the degradation products.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0045740',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of DNA replication',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of DNA replication.',\n",
       "   'synonyms': 'activation of DNA replication, stimulation of DNA replication, up regulation of DNA replication, up-regulation of DNA replication, upregulation of DNA replication',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048010',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'vascular endothelial growth factor receptor signaling pathway',\n",
       "   'definition': 'Any series of molecular signals initiated by the binding of an extracellular ligand to a vascular endothelial growth factor receptor (VEGFR) located on the surface of the receiving cell, and ending with regulation of a downstream cellular process, e.g. transcription.',\n",
       "   'synonyms': 'VEGF receptor signaling pathway, VEGF receptor signalling pathway, VEGFR signaling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048012',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'hepatocyte growth factor receptor signaling pathway',\n",
       "   'definition': 'The series of molecular signals generated as a consequence of the hepatocyte growth factor receptor binding to one of its physiological ligands.',\n",
       "   'synonyms': 'HGF receptor signaling pathway, HGF receptor signalling pathway, Met signaling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048013',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'ephrin receptor signaling pathway',\n",
       "   'definition': 'The series of molecular signals generated as a consequence of an ephrin receptor binding to an ephrin.',\n",
       "   'synonyms': 'Eph receptor signaling pathway, Eph receptor signalling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048261',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'negative regulation of receptor-mediated endocytosis',\n",
       "   'definition': 'Any process that stops, prevents, or reduces the frequency, rate or extent of receptor mediated endocytosis, the uptake of external materials by cells, utilizing receptors to ensure specificity of transport.',\n",
       "   'synonyms': 'down regulation of receptor mediated endocytosis, down-regulation of receptor mediated endocytosis, downregulation of receptor mediated endocytosis, inhibition of receptor mediated endocytosis, negative regulation of receptor mediated endocytosis',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0048870',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell motility',\n",
       "   'definition': 'Any process involved in the controlled self-propelled movement of a cell that results in translocation of the cell from one place to another.',\n",
       "   'synonyms': 'cell locomotion, cell movement, movement of a cell',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0050690',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of defense response to virus by virus',\n",
       "   'definition': 'Any viral process that modulates the frequency, rate, or extent of the antiviral response of the host cell or organism.',\n",
       "   'synonyms': 'regulation by virus of antiviral response, regulation of antiviral response by virus, viral regulation of antiviral response',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051056',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of small GTPase mediated signal transduction',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of small GTPase mediated signal transduction.',\n",
       "   'synonyms': 'regulation of small GTPase-mediated signal transduction',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051492',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of stress fiber assembly',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of the assembly of a stress fiber, a bundle of microfilaments and other proteins found in fibroblasts.',\n",
       "   'synonyms': 'regulation of stress fibre biosynthesis, regulation of stress fibre formation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051496',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of stress fiber assembly',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of the assembly of a stress fiber, a bundle of microfilaments and other proteins found in fibroblasts.',\n",
       "   'synonyms': 'activation of stress fiber formation, positive regulation of stress fibre biosynthesis, positive regulation of stress fibre formation, stimulation of stress fiber formation, up regulation of stress fiber formation, up-regulation of stress fiber formation, upregulation of stress fiber formation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051668',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'localization within membrane',\n",
       "   'definition': 'Any process in which a substance or cellular entity, such as a protein complex or organelle, is transported to, and/or maintained in, a specific location within a membrane.',\n",
       "   'synonyms': 'establishment and maintenance of localization in membrane, establishment and maintenance of position in membrane, localisation within membrane, localization to membrane, positioning within membrane',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051894',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of focal adhesion assembly',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of focal adhesion assembly, the establishment and maturation of focal adhesions.',\n",
       "   'synonyms': 'activation of focal adhesion formation, stimulation of focal adhesion formation, up regulation of focal adhesion formation, up-regulation of focal adhesion formation, upregulation of focal adhesion formation',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0051897',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of protein kinase B signaling',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of protein kinase B signaling, a series of reactions mediated by the intracellular serine/threonine kinase protein kinase B.',\n",
       "   'synonyms': 'activation of protein kinase B signaling cascade, positive regulation of AKT signaling cascade, positive regulation of AKT signalling cascade, positive regulation of PKB signaling cascade, positive regulation of PKB signalling cascade, positive regulation of protein kinase B signaling cascade, positive regulation of protein kinase B signalling cascade, stimulation of protein kinase B signaling cascade, up regulation of protein kinase B signaling cascade, up-regulation of protein kinase B signaling cascade, upregulation of protein kinase B signaling cascade',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0060071',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Wnt signaling pathway, planar cell polarity pathway',\n",
       "   'definition': 'The series of molecular signals initiated by binding of a Wnt protein to a receptor on the surface of the target cell where activated receptors signal via downstream effectors including C-Jun N-terminal kinase (JNK) to modulate cytoskeletal elements and control cell polarity.',\n",
       "   'synonyms': 'PCP pathway, Wnt receptor signaling pathway, planar cell polarity pathway, Wnt receptor signalling pathway, planar cell polarity pathway, Wnt-JNK signaling pathway, Wnt-PCP signaling pathway, Wnt-activated signaling pathway, planar cell polarity pathway, non-canonical Wnt signaling pathway, planar cell polarity pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0060263',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of respiratory burst',\n",
       "   'definition': 'Any process that modulates the rate frequency or extent of a phase of elevated metabolic activity, during which oxygen consumption increases; this leads to the production, by an NADH dependent system, of hydrogen peroxide (H2O2), superoxide anions and hydroxyl radicals.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0071260',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cellular response to mechanical stimulus',\n",
       "   'definition': 'Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a mechanical stimulus.',\n",
       "   'synonyms': 'cellular mechanical stimulus response',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0071526',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'semaphorin-plexin signaling pathway',\n",
       "   'definition': 'A series of molecular signals generated as a consequence of a semaphorin receptor (composed of a plexin and a neurophilin) binding to a semaphorin ligand.',\n",
       "   'synonyms': 'semaphorin-plexin signalling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0090023',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of neutrophil chemotaxis',\n",
       "   'definition': 'Any process that increases the frequency, rate, or extent of neutrophil chemotaxis. Neutrophil chemotaxis is the directed movement of a neutrophil cell, the most numerous polymorphonuclear leukocyte found in the blood, in response to an external stimulus, usually an infection or wounding.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0097178',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'ruffle assembly',\n",
       "   'definition': 'The aggregation, arrangement and bonding together of a set of components to form a ruffle, a projection at the leading edge of a crawling cell; the protrusions are supported by a microfilament meshwork. The formation of ruffles (also called membrane ruffling) is thought to be controlled by a group of enzymes known as Rho GTPases, specifically RhoA, Rac1 and cdc42.',\n",
       "   'synonyms': 'membrane ruffle formation, membrane ruffling',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:1900026',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'positive regulation of substrate adhesion-dependent cell spreading',\n",
       "   'definition': 'Any process that activates or increases the frequency, rate or extent of substrate adhesion-dependent cell spreading.',\n",
       "   'synonyms': 'positive regulation of cell spreading during cell substrate adhesion, positive regulation of substrate adhesion dependent cell spreading, up regulation of cell spreading during cell substrate adhesion, up regulation of substrate adhesion dependent cell spreading, up regulation of substrate adhesion-dependent cell spreading',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:1902622',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'regulation of neutrophil migration',\n",
       "   'definition': 'Any process that modulates the frequency, rate or extent of neutrophil migration.',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0000139',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Golgi membrane',\n",
       "   'definition': 'The lipid bilayer surrounding any of the compartments of the Golgi apparatus.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005737',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cytoplasm',\n",
       "   'definition': 'All of the contents of a cell excluding the plasma membrane and nucleus, but including other subcellular structures.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005789',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'endoplasmic reticulum membrane',\n",
       "   'definition': 'The lipid bilayer surrounding the endoplasmic reticulum.',\n",
       "   'synonyms': 'ER membrane',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005802',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'trans-Golgi network',\n",
       "   'definition': 'The network of interconnected tubular and cisternal structures located within the Golgi apparatus on the side distal to the endoplasmic reticulum, from which secretory vesicles emerge. The trans-Golgi network is important in the later stages of protein secretion where it is thought to play a key role in the sorting and targeting of secreted proteins to the correct destination.',\n",
       "   'synonyms': 'Golgi trans face, Golgi trans-face, TGN, late Golgi, maturing face, trans Golgi network, trans face',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005829',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cytosol',\n",
       "   'definition': 'The part of the cytoplasm that does not contain organelles but which does contain other particulate matter, such as protein complexes.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005856',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cytoskeleton',\n",
       "   'definition': 'Any of the various filamentous elements that form the internal framework of cells, and typically remain after treatment of the cells with mild detergent to remove membrane constituents and soluble components of the cytoplasm. The term embraces intermediate filaments, microfilaments, microtubules, the microtrabecular lattice, and other structures characterized by a polymeric filamentous nature and long-range order within the cell. The various elements of the cytoskeleton not only serve in the maintenance of cellular shape but also have roles in other cellular functions, including cellular movement, cell division, endocytosis, and movement of organelles.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005884',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'actin filament',\n",
       "   'definition': 'A filamentous structure formed of a two-stranded helical polymer of the protein actin and associated proteins. Actin filaments are a major component of the contractile apparatus of skeletal muscle and the microfilaments of the cytoskeleton of eukaryotic cells. The filaments, comprising polymerized globular actin molecules, appear as flexible structures with a diameter of 5-9 nm. They are organized into a variety of linear bundles, two-dimensional networks, and three dimensional gels. In the cytoskeleton they are most highly concentrated in the cortex of the cell just beneath the plasma membrane.',\n",
       "   'synonyms': 'microfilament',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005886',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'plasma membrane',\n",
       "   'definition': 'The membrane surrounding a cell that separates the cell from its external environment. It consists of a phospholipid bilayer and associated proteins.',\n",
       "   'synonyms': 'bacterial inner membrane, cell membrane, cellular membrane, cytoplasmic membrane, inner endospore membrane, juxtamembrane, plasma membrane lipid bilayer, plasmalemma',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005925',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'focal adhesion',\n",
       "   'definition': 'Small region on the surface of a cell that anchors the cell to the extracellular matrix and that forms a point of termination of actin filaments.',\n",
       "   'synonyms': 'adhesion plaque, focal contact, hemi-adherens junction',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0005938',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell cortex',\n",
       "   'definition': 'The region of a cell that lies just beneath the plasma membrane and often, but not always, contains a network of actin filaments and associated proteins.',\n",
       "   'synonyms': 'cell periphery, peripheral cytoplasm',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0016020',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'membrane',\n",
       "   'definition': 'A lipid bilayer along with all the proteins and protein complexes embedded in it an attached to it.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0030027',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'lamellipodium',\n",
       "   'definition': 'A thin sheetlike process extended by the leading edge of a migrating cell or extending cell process; contains a dense meshwork of actin filaments.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0030667',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'secretory granule membrane',\n",
       "   'definition': 'The lipid bilayer surrounding a secretory granule.',\n",
       "   'synonyms': 'secretory vesicle membrane',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0031410',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cytoplasmic vesicle',\n",
       "   'definition': 'A vesicle found in the cytoplasm of a cell.',\n",
       "   'synonyms': 'cytoplasmic membrane bounded vesicle, cytoplasmic membrane-enclosed vesicle, cytoplasmic, membrane-bounded vesicle',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0032587',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'ruffle membrane',\n",
       "   'definition': 'The portion of the plasma membrane surrounding a ruffle.',\n",
       "   'synonyms': 'membrane ruffle',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0036464',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cytoplasmic ribonucleoprotein granule',\n",
       "   'definition': 'A ribonucleoprotein granule located in the cytoplasm.',\n",
       "   'synonyms': 'Staufen granule',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0042470',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'melanosome',\n",
       "   'definition': 'A tissue-specific, membrane-bounded cytoplasmic organelle within which melanin pigments are synthesized and stored. Melanosomes are synthesized in melanocyte cells.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0042995',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'cell projection',\n",
       "   'definition': 'A prolongation or process extending from a cell, e.g. a flagellum or axon.',\n",
       "   'synonyms': 'cell process, cellular process, cellular projection',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0043197',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'dendritic spine',\n",
       "   'definition': 'A small, membranous protrusion from a dendrite that forms a postsynaptic compartment - typically receiving input from a single presynapse.  They function as partially isolated biochemical and an electrical compartments. Spine morphology is variable including \"thin\", \"stubby\", \"mushroom\", and \"branched\", with a continuum of intermediate morphologies. They typically terminate in a bulb shape, linked to the dendritic shaft by a restriction. Spine remodeling is though to be involved in synaptic plasticity.',\n",
       "   'synonyms': 'branched dendritic spine, dendrite spine, mushroom dendritic spine, sessile dendritic spine, stubby dendritic spine, thin dendritic spine',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0043231',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'intracellular membrane-bounded organelle',\n",
       "   'definition': 'Organized structure of distinctive morphology and function, bounded by a single or double lipid bilayer membrane and occurring within the cell. Includes the nucleus, mitochondria, plastids, vacuoles, and vesicles. Excludes the plasma membrane.',\n",
       "   'synonyms': 'intracellular membrane-enclosed organelle',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0055038',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'recycling endosome membrane',\n",
       "   'definition': 'The lipid bilayer surrounding a recycling endosome.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0070062',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'extracellular exosome',\n",
       "   'definition': 'A vesicle that is released into the extracellular region by fusion of the limiting endosomal membrane of a multivesicular body with the plasma membrane. Extracellular exosomes, also simply called exosomes, have a diameter of about 40-100 nm.',\n",
       "   'synonyms': 'exosome, extracellular vesicular exosome',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0098794',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'postsynapse',\n",
       "   'definition': 'The part of a synapse that is part of the post-synaptic cell.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0098978',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'glutamatergic synapse',\n",
       "   'definition': 'A synapse that uses glutamate as a neurotransmitter.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0101003',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'ficolin-1-rich granule membrane',\n",
       "   'definition': 'The lipid bilayer surrounding a ficolin-1-rich granule.',\n",
       "   'ontology': 'C'}},\n",
       " {'id': 'GO:0000166',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'nucleotide binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with a nucleotide, any compound consisting of a nucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the ribose or deoxyribose.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0003924',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'GTPase activity',\n",
       "   'definition': 'Catalysis of the reaction: GTP + H2O = GDP + phosphate.',\n",
       "   'synonyms': 'ARF small monomeric GTPase activity, RHEB small monomeric GTPase activity, Rab small monomeric GTPase activity, Ran small monomeric GTPase activity, Ras small monomeric GTPase activity, Rho small monomeric GTPase activity, Sar small monomeric GTPase activity, dynamin GTPase activity, heterotrimeric G-protein GTPase activity, heterotrimeric G-protein GTPase, alpha-subunit, heterotrimeric G-protein GTPase, beta-subunit, heterotrimeric G-protein GTPase, gamma-subunit, hydrolase activity, acting on acid anhydrides, acting on GTP, involved in cellular and subcellular movement, protein-synthesizing GTPase activity, protein-synthesizing GTPase activity, elongation, protein-synthesizing GTPase activity, initiation, protein-synthesizing GTPase activity, termination, signal-recognition-particle GTPase activity, small monomeric GTPase activity, tubulin GTPase activity',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0005515',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'protein binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules).',\n",
       "   'synonyms': 'glycoprotein binding, protein amino acid binding',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0005525',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'GTP binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with GTP, guanosine triphosphate.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0016787',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'hydrolase activity',\n",
       "   'definition': 'Catalysis of the hydrolysis of various bonds, e.g. C-O, C-N, C-C, phosphoric anhydride bonds, etc. Hydrolase is the systematic name for any enzyme of EC class 3.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0017137',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Rab GTPase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with Rab protein, any member of the Rab subfamily of the Ras superfamily of monomeric GTPases.',\n",
       "   'synonyms': 'REP, Rab escort protein activity, Rab interactor activity',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0019899',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'enzyme binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with any enzyme.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0019901',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'protein kinase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with a protein kinase, any enzyme that catalyzes the transfer of a phosphate group, usually from ATP, to a protein substrate.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0031996',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'thioesterase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with any thioesterase enzyme.',\n",
       "   'synonyms': 'thiolesterase binding',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0042826',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'histone deacetylase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with the enzyme histone deacetylase.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0044877',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'protein-containing complex binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with a macromolecular complex.',\n",
       "   'synonyms': 'macromolecular complex binding, protein complex binding',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0051022',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'Rho GDP-dissociation inhibitor binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with a Rho GDP-dissociation inhibitor protein.',\n",
       "   'synonyms': 'Rho GDI binding',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0051117',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'A',\n",
       "  'detail': {'name': 'ATPase binding',\n",
       "   'definition': 'Interacting selectively and non-covalently with an ATPase, any enzyme that catalyzes the hydrolysis of ATP.',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0006629',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'lipid metabolic process',\n",
       "   'definition': 'The chemical reactions and pathways involving lipids, compounds soluble in an organic solvent but not, or sparingly, in an aqueous solvent. Includes fatty acids; neutral fats, other fatty-acid esters, and soaps; long-chain (fatty) alcohols and waxes; sphingoids and other long-chain bases; glycolipids, phospholipids and sphingolipids; and carotenes, polyprenols, sterols, terpenes and other isoprenoids.',\n",
       "   'synonyms': 'lipid metabolism',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0007165',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'signal transduction',\n",
       "   'definition': 'The cellular process in which a signal is conveyed to trigger a change in the activity or state of a cell. Signal transduction begins with reception of a signal (e.g. a ligand binding to a receptor or receptor activation by a stimulus such as light), or for signal transduction in the absence of ligand, signal-withdrawal or the activity of a constitutively active receptor. Signal transduction ends with regulation of a downstream cellular process, e.g. regulation of transcription or regulation of a metabolic process. Signal transduction covers signaling from receptors located on the surface of the cell and signaling via molecules located within the cell. For signaling between cells, signal transduction is restricted to events at and within the receiving cell.',\n",
       "   'synonyms': 'signaling cascade, signaling pathway, signalling cascade, signalling pathway',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0035556',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'intracellular signal transduction',\n",
       "   'definition': 'The process in which a signal is passed on to downstream components within the cell, which become activated themselves to further propagate the signal and finally trigger a change in the function or state of the cell.',\n",
       "   'synonyms': 'intracellular protein kinase cascade, intracellular signal transduction pathway, intracellular signaling cascade, intracellular signaling chain, intracellular signaling pathway, protein kinase cascade, signal transduction via intracellular signaling cascade, signal transmission via intracellular cascade',\n",
       "   'ontology': 'B'}},\n",
       " {'id': 'GO:0004435',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'phosphatidylinositol phospholipase C activity',\n",
       "   'definition': 'Catalysis of the reaction: 1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate + H(2)O = 1,2-diacylglycerol + 1D-myo-inositol 1,4,5-trisphosphate + H(+).',\n",
       "   'synonyms': '1-phosphatidyl-1D-myo-inositol-4,5-bisphosphate inositoltrisphosphohydrolase activity, 1-phosphatidyl-D-myo-inositol-4,5-bisphosphate inositoltrisphosphohydrolase activity, 1-phosphatidylinositol-4,5-bisphosphate phosphodiesterase activity, PI-PLC activity, monophosphatidylinositol phosphodiesterase activity, phosphatidylinositol-4,5-bisphosphate hydrolysis, phosphoinositidase C activity, phosphoinositide phospholipase C activity, triphosphoinositide phosphodiesterase activity',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0004629',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'phospholipase C activity',\n",
       "   'definition': 'Catalysis of the reaction: a phospholipid + H2O = 1,2-diacylglycerol + a phosphatidate.',\n",
       "   'synonyms': 'lecithinase C activity, lipophosphodiesterase C, phosphatidase C',\n",
       "   'ontology': 'F'}},\n",
       " {'id': 'GO:0008081',\n",
       "  'structureId': '2FJU',\n",
       "  'chainId': 'B',\n",
       "  'detail': {'name': 'phosphoric diester hydrolase activity',\n",
       "   'definition': 'Catalysis of the hydrolysis of a phosphodiester to give a phosphomonoester and a free hydroxyl group.',\n",
       "   'synonyms': 'phosphodiesterase',\n",
       "   'ontology': 'F'}}]"
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
